src/main/java/de/zalando/ep/zalenium/container/DockerContainerClient.java

```diff
@@ -229,12 +213,21 @@ public int getRunningContainers(String image) {
         return 0;
     }
 
-    public ContainerCreationStatus createContainer(String zaleniumContainerName, String image, Map<String, String> envVars,
+    @Override
+    public ContainerCreationStatus createContainer(String zaleniumContainerName, String image, Map<String, String> envVars, String nodePort) {
+        return createContainer(zaleniumContainerName, image, DEFAULT_SELENIUM_CONTAINER_CPU_LIMIT, DEFAULT_SELENIUM_CONTAINER_MEMORY_LIMIT, envVars, nodePort, NAME_COLLISION_RETRIES);
+    }
+
+    public ContainerCreationStatus createContainer(String zaleniumContainerName, String image,
```

Comment:
There is no need to change the method signature, just grab the environment variables inside it, either by calling a method or just with a couple of code lines.